jax_healpy/sphtfunc.py

```diff
@@ -55,6 +69,297 @@ def deferred_func(*args: Param.args, **kwargs: Param.kwargs) -> ReturnType:
     return deferred_func
 
 
+def _compute_beam_window(lmax: int, fwhm: float = 0.0, sigma: float | None = None) -> ArrayLike:
+    """Return Gaussian beam factors up to lmax."""
+    if sigma is None:
+        # Use jnp.where to handle fwhm=0 case in a JAX-compatible way
+        sigma = jnp.where(fwhm == 0.0, 0.0, fwhm / jnp.sqrt(8.0 * jnp.log(2.0)))
```

Comment:
I don't think it's necessary

```suggestion
        sigma = fwhm / jnp.sqrt(8.0 * jnp.log(2.0))
```